texar/torch/modules/classifiers/conv_classifiers.py

```diff
@@ -163,7 +163,8 @@ def default_hparams() -> Dict[str, Any]:
 
     def forward(self,  # type:ignore
                 input: torch.Tensor,
-                sequence_length: Union[torch.LongTensor, List[int]] = None,
+                sequence_length: Optional[Union[torch.LongTensor,
+                                                List[int]]] = None,
```

Comment:
We don't allow `sequence_length` to be `List[int]` in some methods (such as the one above, in `losses/rewards.py` `discount_reward`). Let's keep the behavior consistent. If we decide that `sequence_length` can be `List[int]` everywhere, we might as well add a type alias for it and add some utility functions to transform it into `LongTensor`.